Description: Fix build on armel without armv5te support.
 libavutil/arm/asm.S sets '.arch' depending on HAVE_ARMV5TE so that
 assembling armv5te code will always succeed even if the default -march
 flag does not support it. HAVE_ARMV5TE_EXTERNAL tests assembling code
 with the default arch.
 .
 Fixes the missing symbol ff_prefetch_arm with --cpu= not including
 armv5te.
Origin: upstream, https://patches.libav.org/patch/33688/
Author: Janne Grunau <janne-libav@jannau.net>
Forwarded: yes
Last-Update: 2013-01-20

--- a/libavcodec/arm/videodsp_armv5te.S
+++ b/libavcodec/arm/videodsp_armv5te.S
@@ -22,7 +22,6 @@
 #include "config.h"
 #include "libavutil/arm/asm.S"
 
-#if HAVE_ARMV5TE_EXTERNAL
 function ff_prefetch_arm, export=1
         subs            r2,  r2,  #1
         pld             [r0]
@@ -30,4 +29,3 @@
         bne             ff_prefetch_arm
         bx              lr
 endfunc
-#endif
